src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java

```diff
@@ -41,7 +41,7 @@ public class FFmpegLogCallback extends LogCallback {
 
     static final FFmpegLogCallback instance = new FFmpegLogCallback().retainReference();
 
-    /** Returns an instance that can be used with {@link #setLogCallback(LogCallback)}. */
+    /** Returns an instance that can be used with {@link org.bytedeco.ffmpeg.global.avutil#setLogCallback(LogCallback)}. */
```

Comment:
Just `{@link avutil#setLogCallback(LogCallback)}` should work, no?

Reply:
Nope, it doesn't:

```
/Users/xxx/Projects/javacv/src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44: warning: Tag @link: reference not found: avutil#setLogCallback(LogCallback)
    /** Returns an instance that can be used with {@link avutil#setLogCallback(LogCallback)}. */
```

Reply:
Ok, anything else you might want to fix, or can we merge this?

Reply:
I think we're good on that :)